### PR TITLE
Added 8 Links

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Issue #71

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -788,6 +792,8 @@ <h1>Featured Speakers</h1>
 											<a target="default" href="http://datacharmer.org/"><i class="icon fa fa-external-link"></i></a>
                                              <a target="default" href="https://github.com/datacharmer"><i class="icon fa fa-github"></i></a>											<a target="default" href="https://twitter.com/yacy_search"><i class="icon social_twitter"></i></a>
 											<a target="default" href="https://www.linkedin.com/in/datacharmer/"><i class="icon social_linkedin"></i></a>
```
Comment (0x48piraj) [inline index.html]: Indentation !

**Review by 0x48piraj — COMMENTED** [review]
Provide a deploy link.

**Comment by arjunpandey2003** [comment]
https://arjunpandey2003.github.io/2017.arjunpfossasia.org/.

**Comment by supunTE** [comment]
@arjunpandey2003 please describe your changes

File: index.html [hunk 2]
```diff
@@ -601,6 +601,7 @@ <h1>Featured Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="default" href="https://twitter.com/0rb1t3r"><i class="icon social_twitter"></i></a>
 											<a target="default" href="https://www.linkedin.com/in/orbiter/"><i class="icon social_linkedin"></i></a>
+                                                                                        <a target="default" href="https://github.com/Orbiter"><i class="icon fa fa-github"></i></a>
```
Comment (supunTE) [inline index.html:608]: Indent

File: index.html [hunk 3]
```diff
@@ -649,9 +650,14 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://metafluff.com/"><i class="icon fa fa-external-link"></i></a>
+
+                                                                     			<a target="default" href="https://github.com/autonome"><i class="icon fa fa-github"></i></a>
```
Comment (supunTE) [inline index.html]: This line already there

Comment (supunTE) [reply index.html]: See line 656

File: index.html [hunk 4]
```diff
@@ -649,9 +650,14 @@ <h1>Featured Speakers</h1>
...
+
```
Comment (supunTE) [inline index.html]: Remove unwanted spaces

File: index.html [hunk 5]
```diff
@@ -649,9 +650,14 @@ <h1>Featured Speakers</h1>
...
                                             						<a target="default" href="https://github.com/autonome"><i class="icon fa fa-github"></i></a>
```
Comment (supunTE) [inline index.html]: see 654 line.

File: index.html [hunk 6]
```diff
@@ -649,9 +650,14 @@ <h1>Featured Speakers</h1>
...
+
 											<a target="default" href="https://twitter.com/dietrich?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"><i class="icon social_twitter"></i></a>
 											<a target="default" href="https://www.linkedin.com/in/dietrich/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://github.com/autonome"><i class="icon fa fa-github"></i></a>
```
Comment (supunTE) [inline index.html]: Indent

**Review by supunTE — CHANGES_REQUESTED** [review]

File: index.html [hunk 7]
```diff
@@ -717,7 +722,11 @@ <h1>Featured Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="default" href="http://www.andyselle.com"><i class="icon fa fa-external-link"></i></a>
 											<a target="default" href="https://research.google.com/teams/brain/"><i class="icon fa fa-external-link"></i></a>
+
+                                                                                        <a target="default" href="http://github.com/aselle"><i class="icon fa fa-github"></i></a>
+
                                             						<a target="default" href="http://github.com/aselle"><i class="icon fa fa-github"></i></a>
```
Comment (supunTE) [inline index.html]: This line already there see line 726!

File: index.html [hunk 8]
```diff
@@ -751,7 +760,11 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://twitter.com/simon_riggs"><i class="icon social_twitter"></i></a>
+
```
Comment (supunTE) [inline index.html]: Remove unwanted spaces

File: index.html [hunk 9]
```diff
@@ -751,7 +760,11 @@ <h1>Featured Speakers</h1>
...
+											<a target="default" href="https://2ndquadrant.com"><i class="icon fa fa-external-link"></i></a>
+
 											<a target="default" href="https://2ndquadrant.com"><i class="icon fa fa-external-link">
```
Comment (supunTE) [inline index.html]: This line already there see 764!

**Review by supunTE — CHANGES_REQUESTED** [review]

File: index.html [hunk 10]
```diff
@@ -751,7 +760,11 @@ <h1>Featured Speakers</h1>
...
+
```
Comment (supunTE) [inline index.html]: Remove unwanted spaces

File: index.html [hunk 11]
```diff
@@ -770,6 +783,11 @@ <h1>Featured Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="default" href="https://twitter.com/fweng322"><i class="icon social_twitter"></i></a>
 											<a target="default" href="http://www.slat.org/"><i class="icon fa fa-external-link"></i></a>
+
+                                                                                        <a target="default" href="https://github.com/fweng322"><i class="icon fa fa-github"></i></a>
+                                                                                        <a target="default" href="https://fosstodon.org/@franklin"><i class="icon fa fa-external-link"></i></a>
+	
+
```
Comment (supunTE) [inline index.html]: Remove unwanted spaces

File: index.html [hunk 12]
```diff
@@ -787,8 +805,13 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://datacharmer.org/"><i class="icon fa fa-external-link"></i></a>
+
+                                                                                        <a target="default" href="https://github.com/datacharmer"><i class="icon fa fa-github"></i></a>											<a target="default" href="https://twitter.com/yacy_search"><i class="icon social_twitter"></i></a>
+
                                             						<a target="default" href="https://github.com/datacharmer"><i class="icon fa fa-github"></i></a>											<a target="default" href="https://twitter.com/yacy_search"><i class="icon social_twitter"></i></a>
-											<a target="default" href="https://www.linkedin.com/in/datacharmer/"><i class="icon social_linkedin"></i></a>
+			<a target="default" href="https://www.linkedin.com/in/datacharmer/"><i class="icon social_linkedin"></i></a>
```
Comment (supunTE) [inline index.html]: indent

File: index.html [hunk 13]
```diff
@@ -787,8 +805,13 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://datacharmer.org/"><i class="icon fa fa-external-link"></i></a>
+
+                                                                                        <a target="default" href="https://github.com/datacharmer"><i class="icon fa fa-github"></i></a>											<a target="default" href="https://twitter.com/yacy_search"><i class="icon social_twitter"></i></a>
```
Comment (supunTE) [inline index.html]: This line already there!

File: index.html [hunk 14]
```diff
@@ -805,11 +828,12 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://cosmiqo.com/"><i class="icon fa fa-external-link"></i></a>
-                                            <a target="default" href="https://github.com/melvinzhang"><i class="icon fa fa-github"></i></a>
+                      <a target="default" href="https://github.com/melvinzhang"><i class="icon fa fa-github"></i></a>
 											<a target="default" href="https://twitter.com/melvinzhangzy"><i class="icon social_twitter"></i></a>
```
Comment (supunTE) [inline index.html]: Indent

File: index.html [hunk 15]
```diff
@@ -823,9 +847,10 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://nrnjn.xyz/"><i class="icon fa fa-external-link"></i></a>
-                                            <a target="default" href="https://github.com/niranjan94"><i class="icon fa fa-github"></i></a>
+                      <a target="default" href="https://github.com/niranjan94"><i class="icon fa fa-github"></i></a>
 											<a target="default" href="https://twitter.com/niranjan94"><i class="icon social_twitter"></i></a>
```
Comment (supunTE) [inline index.html:839]: Indent

File: index.html [hunk 16]
```diff
@@ -875,8 +899,7 @@ <h1>Featured Speakers</h1>
 									<img class="background-image" alt="Yuen Hoe Lim" src="img/YuenHoeLim.jpg">
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
-											<a target="default" href="https://yuenhoe.com/"><i class="icon fa fa-external-link"></i></a>
-                                            <a target="default" href="https://github.com/jasonmoofang/"><i class="icon fa fa-github"></i></a>
+											<a target="default" href="https://yuenhoe.com/"> <a target="default" href="https://github.com/jasonmoofang/"><i class="icon fa fa-github"></i></a>
```
Comment (supunTE) [inline index.html:891]: Indent!

**Review by supunTE — CHANGES_REQUESTED** [review]

File: index.html [hunk 17]
```diff
@@ -770,6 +774,8 @@ <h1>Featured Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="default" href="https://twitter.com/fweng322"><i class="icon social_twitter"></i></a>
 											<a target="default" href="http://www.slat.org/"><i class="icon fa fa-external-link"></i></a>
+                                                                                        <a target="default" href="https://github.com/fweng322"><i class="icon fa fa-github"></i></a>
```
Comment (supunTE) [inline index.html]: Indent needed!

File: index.html [hunk 18]
```diff
@@ -717,6 +719,7 @@ <h1>Featured Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="default" href="http://www.andyselle.com"><i class="icon fa fa-external-link"></i></a>
 											<a target="default" href="https://research.google.com/teams/brain/"><i class="icon fa fa-external-link"></i></a>
+                                                                                        <a target="default" href="http://github.com/aselle"><i class="icon fa fa-github"></i></a>
```
Comment (supunTE) [inline index.html]: small indent

**Review by supunTE — CHANGES_REQUESTED** [review]
Change those

**Comment by supunTE** [comment]
Please can you squash your commits.

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Resolve conflicts

File: index.html [hunk 19]
```diff
@@ -649,7 +650,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="http://metafluff.com/"><i class="icon fa fa-external-link"></i></a>
-                                            						<a target="default" href="https://github.com/autonome"><i class="icon fa fa-github"></i></a>
+                                                                     			<a target="default" href="https://github.com/autonome"><i class="icon fa fa-github"></i></a>
```
Comment (abishekvashok) [inline index.html:659]: You got unwanted tabs at the end.

**Comment by arjunpandey2003** [comment]
Resolved

File: index.html [hunk 20]
```diff
@@ -717,7 +717,7 @@ <h1>Featured Speakers</h1>
 										<div class="social-links vertical-align">
 											<a target="default" href="http://www.andyselle.com"><i class="icon fa fa-external-link"></i></a>
 											<a target="default" href="https://research.google.com/teams/brain/"><i class="icon fa fa-external-link"></i></a>
-                                            						<a target="default" href="http://github.com/aselle"><i class="icon fa fa-github"></i></a>
+                                                                                        <a target="default" href="http://github.com/aselle"><i class="icon fa fa-github"></i></a>
```
Comment (abishekvashok) [inline index.html]: Same here

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by vaibhavsingh97** [comment]
Please resolve conflicts

**Comment by samyakbvs** [comment]
Great job! @arjunpandey2003

**Review by samyakbvs — APPROVED** [review]

**Review by supunTE — APPROVED** [review]
LGTM! If you merge it and squash it will be excellent.

**Comment by arjunpandey2003** [comment]
@mariobehling please review

**Comment by mariobehling** [comment]
Your test page shows errors. Please check first. Developers are the first testers of their own code. Or if this is not the test page anymore provide a new one, please.

**Comment by arjunpandey2003** [comment]
@mariobehling sorry i dont know what a test page is.

**Review by nzec — COMMENTED** [review]
Merge Conflicts!

**Pull request closed without merging**